hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java

```diff
@@ -630,13 +630,15 @@ private void handleLogLevelReset(HttpPostCommand command) throws UnsupportedEnco
 
     protected void handleConfigReload(HttpPostCommand command) throws UnsupportedEncodingException {
         prepareResponse(
+                SC_500,
```

Comment:
How about HTTP 402 Payment required?

Reply:
Agree 402 is better, please see my comment
https://github.com/hazelcast/hazelcast/pull/20737#discussion_r808022587